Makefile

```diff
@@ -28,3 +29,6 @@ legacy: dist/stable/styles/ $(LEGACY_CSS_OUTPUTS)
 # Utility rules
 clean:
 	rm -rf dist
+
+rebuildcss:
+	rm -rf dist/css && make css
```

Comment:
Couldn't `make -B css` work?

If we do keep this I think it should be `cleanbuild` (but I guess since it's only CSS then `cleanbuildcss`?)